display/ng.c

```diff
@@ -123,7 +122,10 @@ int
 ng_init(void *dev, int debug)
 {
   /* Don't change this number. */
-  assert (DISPLAYS == 8);

```

Comment:
This seems like a good place for STATIC_ASSERT.  Having a compile time invariant checked at runtime is not the best answer.

Reply:
assert() is a runtime functionality only on DEBUG builds and otherwise does nothing.

The suggested change keeps that effect.

I've changed this to a static check.

The other case was looking at runtime determined data, so a static check wouldn't be useful

This file and the only other place in the codebase where assert() was used doesn't specifically depend on any other simh code.